packages/cdktf-cli/bin/cmds/ui/terraform-context.tsx

```diff
@@ -330,36 +397,66 @@ export const useTerraform = ({ targetDir, synthCommand }: UseTerraformInput) =>
     return state
```

Comment:
Should `planDestroy` also be broken up with a synth check first? I'm not certain the reasons why others were split apart.

Reply:
Hm, in general absolutely yes. Was just wondering why I haven't stumbled upon this. First and foremost certainly because this area lacks tests, but more importantly it's unused code. So, I'm just gonna drop it entirely.

Reply:
6d798dc